pyproject.toml

```diff
@@ -23,6 +23,7 @@ telegraph = "^2.1.0"
 
 [tool.poetry.dev-dependencies]
 black = "^22"
+pydantic = "^1.9.1"
```

Comment:
Why as a dev dependency?

Reply:
TBH, because that was in my history for some reason. Will shift it to normal deps.